corehq/apps/app_manager/tests/test_util.py

```diff
@@ -89,6 +90,7 @@ def _add_scheduler_to_form(self, form, module, form_abreviation):
         phase.add_form(form)
 
 
+@flag_enabled('USER_PROPERTY_EASY_REFS')
```

Comment:
Can you add a test that makes sure user cases don't get sent down if this isn't set